library/build.gradle

```diff
@@ -20,7 +20,7 @@ dependencies {
     compile project(':third_party:disklrucache')
     compile project(':annotation')
     compile "com.android.support:support-v4:${SUPPORT_V4_VERSION}"
-
+    compile "com.android.support:appcompat-v7:${SUPPORT_V7_VERSION}"
```

Comment:
I would prefer to be able to pass an drawable id.  
- Terser calls (no need to initialise the drawable yourself, just pass the id)
- No risk of a crash because you passed the id of a VD to glide (right now it does crash)
- If you refactor a raster to VD, no need to check all the glide calls


Alternatively,

I have not toyed too much with reflection in this kind of use case but I think it might be possible to check for the presence of the class & method from the support lib and use them if they are here.  
If it is not there, fallback to the previous method.

I would also need to add a proguard rule for this class in the library's rules. Although I don't know yet whether I can add a rule for a class that might not be present in the classpath.